admin/tasks/deploy.py

```diff
@@ -292,6 +289,25 @@ def whitelist_check(ctx, email):
     ctx.sudo(f'bash -c "cd {app_dir} && {cmd}"', user=user)
 
 
+@task
+def admin_list(ctx):
+    cmd = f"{venv_dir}/bin/zam_admin production.ini#repondeur list"
+    res = ctx.sudo(f'bash -c "cd {app_dir} && {cmd}"', user=user)
+    return res.stdout
+
+
+@task
+def admin_set(ctx, email):
```

Comment:
Maybe grant / revoke instead of set / unset ?

Reply:
Good idea, I'll do that tomorrow.